front/app/modules/commercial/granular_permissions/admin/components/UserFieldSelection/screens/SelectionScreen.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+
+// components
+import {
+  Box,
+  Title,
+  Text,
+  Button,
+  colors,
+  Badge,
+} from '@citizenlab/cl2-component-library';
+
+// api
+import { IPermissionsCustomFieldData } from 'api/permissions_custom_fields/types';
+
+// services
+import {
+  isBuiltInField,
+  IUserCustomFieldData,
+} from 'services/userCustomFields';
+
+// intl
+import { FormattedMessage } from 'utils/cl-intl';
+import messages from '../../../containers/Granular/messages';
+
+type SelectionScreenProps = {
+  selectedFields: Array<IPermissionsCustomFieldData> | undefined;
+  registrationFieldList: Array<IUserCustomFieldData> | null | undefined;
+  locale: string;
+  handleAddField: (fields: IUserCustomFieldData) => void;
+  setShowAddFieldPage: (show: boolean) => void;
+};
+
+export const SelectionScreen = ({
+  registrationFieldList,
+  selectedFields,
+  locale,
+  handleAddField,
+  setShowAddFieldPage,
+}: SelectionScreenProps) => {
+  return (
+    <>
+      <Box mb="20px">
+        <Title variant="h3" color="primary">
+          <FormattedMessage {...messages.addQuestion} />
```

Comment:
This should be in the header of the modal, now it looks a bit funny: 
![image](https://user-images.githubusercontent.com/13822005/229750498-4b1b4116-07bb-4ca7-8d98-0828374ebf52.png)